ourjapanlife/findadoc-localization
# Add husky and lint

- Set up yarn (so we can run stuff) 🧶 
- Installed husky 🐕  https://typicode.github.io/husky/#/?id=create-a-hook
- Installed eslint with JSON config 🧼 
  - https://eslint.org/docs/user-guide/getting-started
  - https://github.com/azeemba/eslint-plugin-json
  
  
 Now contributors will be reminded to keep their JSON syntax correct on commits.
 
 We should also set up a github action for this in a future PR.
 
 
 To test:
 
1. edit one of the json files to make the json invalid, save
2. run `yarn run lint` => see errors
3. try to commit => see errors
